README.yaml

```diff
@@ -0,0 +1,392 @@
+  - name: Troubleshooting migration tasks in Amazon Database Migration Service
+    description: Troubleshooting migration tasks in Amazon Database Migration Service
+    url: https://docs.amazonaws.cn/en_us/dms/latest/userguide/CHAP_Troubleshooting.html
+  - name: Troubleshoot AWS DMS endpoint connectivity failures
+    description: How can I troubleshoot AWS DMS endpoint connectivity failures?
+    url: https://aws.amazon.com/premiumsupport/knowledge-center/dms-endpoint-connectivity-failures
+  - name: Using PostgreSQL logical replication with Aurora
+    description: Using PostgreSQL logical replication with Aurora
+    url: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Replication.Logical.html
+
+# Short description of this project
+description: |-
+  Terraform modules for provisioning and managing AWS [DMS](https://aws.amazon.com/dms/) resources. 
+  
+  The following DMS resources are supported:
+  
+    - [IAM Roles for DMS](modules/dms-iam)
+    - [DMS Endpoints](modules/dms-endpoint)
+    - [DMS Replication Instances](modules/dms-replication-instance)
+    - [DMS Replication Tasks](modules/dms-replication-task)
+    - [DMS Event Subscriptions](modules/dms-event-subscription)
+
+  Refer to [modules](modules) for more details.
+
+# How to use this module
+usage: |-
+
+  For a complete example, see [examples/complete](examples/complete).
+
+  For automated tests of the example using [bats](https://github.com/bats-core/bats-core) and [Terratest](https://github.com/gruntwork-io/terratest)
+  (which tests and deploys the example on AWS), see [test](test).
+
+# Example usage
+examples: |-
+
+  ```hcl
+    module "dms_iam" {
+      source = "cloudposse/dms/aws//modules/dms-iam"
+      # Cloud Posse recommends pinning every module to a specific version
+      # version     = "x.x.x"
+    
+      context = module.this.context
+    }
+  
+    module "vpc" {
+      source  = "cloudposse/vpc/aws"
+      # Cloud Posse recommends pinning every module to a specific version
+      # version     = "x.x.x"
+    
+      ipv4_primary_cidr_block = "172.19.0.0/16"
+    
+      context = module.this.context
+    }
+  
+    module "subnets" {
+      source  = "cloudposse/dynamic-subnets/aws"
+      # Cloud Posse recommends pinning every module to a specific version
+      # version     = "x.x.x"
+    
+      availability_zones   = ["us-east-2a", "us-east-2b"]
+      vpc_id               = local.vpc_id
+      igw_id               = [module.vpc.igw_id]
+      ipv4_cidr_block      = [module.vpc.vpc_cidr_block]
+      nat_gateway_enabled  = false
+      nat_instance_enabled = false
+    
+      context = module.this.context
+    }
+    
+    module "dms_replication_instance" {
+      source = "cloudposse/dms/aws//modules/dms-replication-instance"
+      # Cloud Posse recommends pinning every module to a specific version
+      # version     = "x.x.x"
+    
+      engine_version               = "3.4.7"
```

Comment:
If auto_minor_version_upgrade is enabled then we should omit the patch version here or terraform will try to revert the db version upon detected drift

Reply:
Id mention this in the engine_version description too so users are aware